hrc_speech_prediction/tests/test_context_model.py

```diff
@@ -7,20 +7,60 @@
 class TestNode(TestCase):
 
     def setUp(self):
+        self.empty_context = []
+        self.context = ['a','b','c']
+        self.contexts = [['a','b','c'],['d','e','f'],['g','h','i'],['a','g','e']]
         pass
 
     def test_empty_has_zero_n_children(self):
-        raise NotImplementedError
+        self.x = Node()
+        self.assertEqual(self.x.n_children, 0)
+
+        #delete this
+        self.x.get_or_add_node([])
+        print("Hello")
+        print(self.x.n_children)
+        #no deletion code
+        # self.x.get_or_add_node(self.contxt)
 
     def test_nonempty_has_n_children(self):
-        raise NotImplementedError
+        self.x = Node()
```

Comment:
Same comment for the `self.x`. Here you typically want to use a local variable. If it makes sense to initialize a Node always the same way for all tests, you can do that but then it should be done in the `setUp` method.